### ISSUE TITLE
We need to expose line direction coordinate as a property

### ISSUE THREAD

**Issue by rniwa** [w3c/selection-api#59]
For block-direction caret movements, we need to expose a property that specifies where the caret movement had started vertically.

**Comment by yosinch** [comment]
Could you explain of meaning "line direction coordinate"?  Did you mean "inline base direction" defined in CSS writing mode spec[1]?

Caret movement and extending selection (moving focus boundary point) should follow "Abstract-to-Physical Mappings" in [1]. This implies change or specify Selection.modify() behavior.

[1] https://drafts.csswg.org/css-writing-modes/

**Comment by rniwa** [comment]
Yes, line direction = x in horizontal writing mode and y in vertical writing mode.
